_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -62,35 +65,37 @@ You will receive a response with the complete RelationshipTemplate.
 }
 ```
 
-## Load the Template and get the Request
+{% include copy-notice description="Save the `truncatedReference` and the `id` of the RelationshipTemplate. You will need them in the next steps." %}
+
+## Load the RelationshipTemplate and get the Request
 
-Now you have to load the Template on the Requestor Connector. You can do so by calling the `POST /api/v2/RelationshipTemplates/Peer` route with the following content. Use the `truncatedReference` you copied before:
+Now you have to [load the RelationshipTemplate on the Recipient Connector]({% link _docs_use-cases/use-case-transport-load-relationship-template-created-by-others.md %}) on the Recipient Connector using the following payload.
 
 ```jsonc
 {
-  "reference": "UkxU..."
+  "reference": "<truncated reference of RelationshipTemplate>"
 }
 ```
 
-If no Relationship exists, this will trigger a process in the enmeshed Runtime. It will create a new incoming Request on which we will work in the next step. You can observe this by long polling the incoming Requests or by waiting for the `consumption.incomingRequestReceived` event.
-
-The long polling is done by calling the `GET /api/v2/Requests/Incoming` route. You can use the query params `source.reference=<id-of-the-template>` and `status=ManualDecisionRequired` to filter for Requests that belong to the Template you are currently working on.
+This will trigger a process in the [enmeshed Runtime]({% link _docs_explore/61-runtime.md %}), which will create a new incoming Request.
+You can observe this by [long polling the incoming Requests]({% link _docs_use-cases/use-case-consumption-query-incoming-requests.md %}) and optionally use the query params `source.reference=<id-of-the-relationshiptemplate>` and `status=ManualDecisionRequired` to filter for Requests that belong to the RelationshipTemplate you are currently working on.
```

Comment:
I'd like "`source.reference=<ID-of-the-RelationshipTemplate>`" better.
Besides, I'm not a big fan of the hyphens. Omitting these would be more consistent with other scenarios.